libimage/pull.go

```diff
@@ -71,6 +72,38 @@ func (r *Runtime) Pull(ctx context.Context, name string, pullPolicy config.PullP
 	if err != nil {
 		return nil, err
 	}
+	rwDirs := func() []string {
+		rw := []string{}
+		for _, d := range []string{
+			r.store.RunRoot(),
+			r.store.GraphRoot(),
+			"/tmp",
+			"/var/tmp",
+			defaultConfig.Engine.ImageCopyTmpDir,
+			defaultConfig.Engine.TmpDir,
+		} {
+			if _, err := os.Stat(d); err == nil {
```

Comment:
Hint: You can use `landlock.RWDirs(...).IgnoreIfMissing()` as a shortcut, if you want.